Ppc64le

sanjay-cpu opened the pull request:
Added support for Travis build on ppc64le.

oschwald commented:
This doesn't seem to interact correctly with the build matrix. It generated a lot of extra ppc64le builds as can be seen on the [build page](https://travis-ci.com/github/sanjay-cpu/libmaxminddb/builds/179400512). It seemed to do a Linux ppc64le build for each item in the build matrix, even the macOS builds.

oschwald commented:
Also, please remove the "Update NOTICE" commits.

sanjay-cpu commented:
@oschwald: Thanks for reply. Now i have updated the .travis.yml correctly. And its generating only one build for ppc64le . and i have removed the NOTICE commits as well. 

Now please merge it.

Thanks,
Sanjay

oschwald commented:
Thanks. I merged from the command line after removing the notice commits.

sanjay-cpu commented:
Thank you so much !!
